### PR TITLE
Fix CQM Major defects in Backend

### PR DESCRIPTION
**Pull request by dayo09**
This commit fixes 4 major defects in modules in Backend directory. It fixes by refering unused objects.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

---

For #1580

### CODE REVIEW

**Comment by dayo09** [comment]
Analysis link: https://analysishub.sec.samsung.net/service/analyses/2218597

File: src/Tests/Backend/Executor.test.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ suite("Backend", function () {
   suite("ExecutorBase", function () {
     suite("#constructor()", function () {
       test("Create dummy executor", function (pass) {
-        new ExecutorBase();
+        const instance = new ExecutorBase();
+        assert.isTrue(instance instanceof ExecutorBase);
 
         pass();
         assert.ok(true);
```
Comment (jyoungyun) [inline src/Tests/Backend/Executor.test.ts:28]: (suggestion)
```suggestion
```

**Review by jyoungyun — DISMISSED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**